test/common/runtime/runtime_impl_test.cc

```diff
@@ -587,6 +587,9 @@ class StaticLoaderImplTest : public LoaderImplTest {
 
 TEST_F(StaticLoaderImplTest, All) {
   setup();
+  const auto& asan_test =
```

Comment:
comment

Reply:
With the default value removed this is not necessary any more.